pulp_rpm/tests/functional/test_advisory.py

```diff
@@ -0,0 +1,449 @@
+import json
```

Comment:
This set of tests is useful, and works - when run on the pulp instance. Because it tests app.advisory.py. Which imports createrepo_c. Which isn't available to the machine the test-harness runs on.

Is there a "right way" to have a test like this run in CI? Or should I just add an @skip, so CI doesn't see it but the test remains to be run manually?

Reply:
Several options.
- You can skip a test depending on conditions (e.g. a successful import).
- You can `pytest.skip` or `raise unittest.Skip` a test that has already started and reached a certain condition.
- You can mark the test `pytest.mark.xfail` (but that is probably not quite accurate here).
- We could introduce a new mark for manual testing (would want to have agreement in a broader group for that).

Reply:
I'll prob take the first option there, good idea. that way we can run it locally, but not fail CI.